[DOCS-12757] Add required service account permissions for SCIM (#36065)
DataDog/documentation: content/en/account_management/scim/_index.md

To avoid losing access to your data, Datadog strongly recommends that you create a [service account][6] dedicated to SCIM. Within that service account, create an application key to use in the SCIM integration.

> The service account must have at minimum the `user_access_invite` and `user_access_manage` permissions. For the full list of required permissions, see the [SCIM API documentation][10].

Reviewer: ```suggestion
The service account requires at minimum the `user_access_invite` and `user_access_manage` permissions. For the full list of required permissions, see the [SCIM API documentation][10].
```
